### PR TITLE
add slow score to pd heartbeat

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

For https://github.com/tikv/tikv/issues/10539 we will add a score to represent the slowness of a tikv store and reports this score to pd through store heartbeat periodically.

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -521,6 +521,8 @@ message StoreStats {
     repeated PeerStat peer_stats = 20;
     // Store query stats
     QueryStats query_stats = 21;
+    // Score that represents the speed of the store, ranges in [1, 100], lower is better.
+    uint64 slow_score = 22;
```
Comment (nolouch) [inline proto/pdpb.proto:525]: May we find a better name? :(

Comment (5kbpers) [reply proto/pdpb.proto:525]: Could you give me some suggestions? 🤣

Comment (nolouch) [reply proto/pdpb.proto:525]: anomaly_score? :cry:

Comment (lhy1024) [reply proto/pdpb.proto:525]: performance_score?

**Review by yiwu-arbug — APPROVED** [review]
LGTM

**Pull request merged**